src/pages/thought-leadership/content/template.html

```diff
@@ -81,11 +81,14 @@ <h3 class="rsTl-relatedTopics-smallHeader">{{ article.title }}</h3>
       <p>
         <a
           href="#"
-          @click.prevent="loadMore"
-          :style="{ 'opacity': visibleContent >= filteredContent.full.length ? .3 : 1 }">
-          Load More Posts
+          v-if="visibleContent < filteredContent.full.length"
+          @click.prevent="loadMore">
+          {{ 'Load More Posts' | translate }}
         </a>
       </p>
     </div>
   </div>
+  <div v-if="!loading && fetchError" class="rsTl-error">
+    {{ fetchError }}
```

Comment:
does fetch error need to be passed into the `| translate` filter?

Reply:
Never seen anyone actually translate an error message. you tell me?